src/Application/Fees/Calculator.php

```diff
@@ -21,19 +21,21 @@ class Calculator
 
     private const string STRIPE_FEE_MAIN_PERCENTAGE_STANDARD = '1.5';
 
+    /** @var string[]   Whole currency unit (e.g. pounds) fee charged *by us* for Stripe credit/debit
```

Comment:
```suggestion
    /** @var string[]   Major currency unit (e.g. pounds) fee charged *by us* for Stripe credit/debit
```

Major / minor seems to be the standard jargon for things like pounds and pence.